templates/containers/kubernetes.html

```diff
@@ -12,7 +12,8 @@
         <h2>Enterprise Kubernetes, anywhere</h2>
         <p>In partnership with Google, Canonical now delivers a &lsquo;pure <abbr title="Kubernetes">K8s</abbr>&rsquo; experience, tested across a wide range of clouds and integrated with modern metrics and monitoring.</p>
         <p>The Canonical Distribution of Kubernetes works across all major public clouds and private infrastructure, enabling your teams to operate Kubernetes clusters on demand, anywhere.</p>
-        <p><a href="/containers/contact-us?product=containers-kubernetes">Talk to us about your container management strategy&nbsp;&rsaquo;</a></p>
+        <p><a href="https://conjure-up.io/" class="p-button--brand"><span class="p-link--external">Try it now&nbsp;</span></a></p>
```

Comment:
I think that the link text should be more explicit. Sometime like Try it now with conjure-up.